pallets/automation-time/src/tests.rs

```diff
@@ -280,29 +355,234 @@ fn schedule_transfer_with_dynamic_dispatch() {
 }
 
 #[test]
-fn calculate_schedule_fee_amount_works() {
+fn calculate_auto_compound_action_schedule_fee_amount_works() {
 	new_test_ext(START_BLOCK_TIME).execute_with(|| {
-		let destination = MultiLocation::new(1, X1(Parachain(PARA_ID)));
-		let delegator_account = AccountId32::new(DELEGATOR_ACCOUNT);
-		let execution_count = 2;
-
-		let action: ActionOf<Test> = Action::XCMP {
-			destination,
-			schedule_fee: destination.into(),
-			execution_fee: AssetPayment { asset_location: destination.into(), amount: 100 },
-			encoded_call: vec![3, 4, 5],
-			encoded_call_weight: Weight::from_ref_time(100_000),
-			overall_weight: Weight::from_ref_time(200_000),
-			schedule_as: Some(delegator_account),
-			flow: InstructionSequence::PayThroughRemoteDerivativeAccount,
+		let num_of_execution = 2;
```

Comment:
This function is important. Could we use an arbitrary number such as 15, instead of 2? 2 is too common.

Reply:
Same applies to the below `num_of_execution` values. Maybe just use a random value from 1 to 20.